docs/get-started/what-is-jina.md

```diff
@@ -1,3 +1,26 @@
+# What is Jina?
+
+Jina is a neural search framework written in Python that helps developers build neural search applications that can be easily scaled and deployed in the Cloud.
+
+It is the part of Jina AI ecosystem that is there to bring your search to production. It is designed to build your applications as a set 
+of independent microservices that can be scaled independently and that used together in a pipeline can solve any search related task. 
+
+Jina gives the power to the developers to build their business logic and helps the user by providing abstractions that will help to modularize, serve and scale your applications in the cloud.
+
+Jina heavily uses other components of Jina Ecosystem as [**DocArray**](https://docarray.jina.ai/) to serve as the main data structure and [**Jina Hub**](https://hub.jina.ai/)  to help share, containerize and reuse 
+components of the search pipelines.
+
+Jina is designed to provide a similar experience when developing your solutions locally and when deploying them int he cloud, helping the developer 
+in every phase of the development cycle.
+
+## Design 
+
+Jina is designed as a lean and easy to use framework, and there are only two main concept to learn:
+
+- {ref}`Executor <executor-cookbook>` is a self-contained component and performs a single task on Documents.
```

Comment:
Here it's weird that we refer to `Documents`, but don't link to them, or mention them as a main component. Maybe link to  `docarray`

Reply:
Done